clear-unused-dns.ts

```diff
@@ -56,18 +96,22 @@ async function deleteTXTRecords(
   await batchDelete(recordsToDelete);
 }
 
-const zones = process.argv.slice(2);
-const apikey = process.env.CLOUDFLARE_API_KEY;
-const email = process.env.CLOUDFLARE_EMAIL;
+const api_token = process.env.LINKUP_CF_API_TOKEN;
+const account_id = process.env.LINKUP_CLOUDFLARE_ACCOUNT_ID;
+const mentimeter_dev_zone_id = process.env.LINKUP_CLOUDFLARE_ZONE_ID;
 
-if (!apikey || !email) {
-  console.error("Missing Cloudflare API key or email");
+if (!api_token || !account_id || !mentimeter_dev_zone_id) {
+  console.error("Missing Cloudflare API Token, Account ID or Zone ID");
   console.error(
-    "Please set CLOUDFLARE_API_KEY and CLOUDFLARE_EMAIL environment variables"
+    "Run `menti localsecrets` to set the required environment variables"
```

Comment:
This doesn't make sense in an open-source repo - but maybe we leave it / it's ok?